app/src/main/java/de/danoeh/antennapod/fragment/NavDrawerFragment.java

```diff
@@ -386,7 +386,7 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                     startActivity(intent);
                 }
             }
-        } else {
+        } else if (UserPreferences.getFeedFilter() != UserPreferences.FEED_FILTER_NONE && navAdapter.showSubscriptionList) {
```

Comment:
Is the line clickable if you do not add the check `navAdapter.showSubscriptionList`?

Reply:
Yes, if you exactly click on it (I think this is only possible on an emulator with a mouse) the filter dialog opens.
I think it is better to make this check before opening the dialog otherwise it could confuse users, who have never heard of this function.

Reply:
Okay :+1: I think there is only one thing left to do: This line is too long (that's also why CI fails)